src/termination.ml

```diff
@@ -1,6 +1,6 @@
 (** Temination checking.
 
-    This module allows the translation of a signature in the TPDB format (used
+    This module allows the translation of a signature in the XTC format (used
     in the termination competition). See the "Termination Problem Database":
     http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB *)
```

Comment:
This is not exactly the URL of the XTC format.

Reply:
But people reading this line probably want to see the Problem Data Base rather than an obscure XSD file.